config/uberon.yml

```diff
@@ -20,18 +20,405 @@ entries:
 
 - exact: /phenoscape-ext.owl
   replacement: http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/edit/phenoscape-ext.owl
+  
+- exact: /composite-vertebrate.owl
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-vertebrate.owl
+  
+- exact: /composite-vertebrate.obo
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-vertebrate.owl
+
+- exact: /composite-metazoan.owl
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-metazoan.owl
+  
+- exact: /composite-metazoan.obo
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-metazoan.obo
 
 - exact: /experimental/ceph.owl
   replacement: http://purl.obolibrary.org/obo/ceph.owl
 
 - prefix: /references/
-  replacement: http://svn.code.sf.net/p/obo/svn/uberon/trunk/reference/
+  replacement: https://raw.githubusercontent.com/obophenotype/uberon/master/reference/
 
 - prefix: /homology/
   replacement: https://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/homology/
 
+- prefix: /releases/2011-09-17/
```

Comment:
This is fine, but what about one rule to match all the whole decade of the 2010s ?

```
- prefix: /releases/201
   replacement: http://svn.code.sf.net/p/obo/svn/uberon/releases/201
```

Since it's a bit obscure it would need comments and tests.

Reply:
Much more elegant, thanks! :) Did that.